meta-lmp-base/classes/lmp.bbclass

```diff
@@ -56,3 +56,66 @@ IMAGE_CMD_ostree_append () {
 	# Update default home path
 	sed -i -e 's,:/home,:/var/rootdirs/home,g' usr/etc/passwd
 }
+
+run_fiotool_cmd () {
+	if [ -n "${SOTA_PACKED_CREDENTIALS}" ]; then
+                if [ -e "${SOTA_PACKED_CREDENTIALS}" ]; then
+                        "${1}" -repo "${OSTREE_REPO}" -creds "${SOTA_PACKED_CREDENTIALS}"
+                else
+                        bbwarn "SOTA_PACKED_CREDENTIALS file does not exist."
+                fi
+        else
+                bbwarn "SOTA_PACKED_CREDENTIALS not set. Please add SOTA_PACKED_CREDENTIALS."
+        fi
+
+}
+
+do_image_ostreepush[depends] += "fiotools-native:do_populate_sysroot ca-certificates-native:do_populate_sysroot"
```

Comment:
Looking at the final generated output, we just need to add fiotools-native really, as ca-certificates will be set by meta-updater.